resources/tests.mjs

```diff
@@ -1,9 +1,16 @@
 import { BenchmarkTestStep } from "./benchmark-runner.mjs";
 import { todos } from "./translations.mjs";
+import { params } from "./shared/params.mjs";
 
-const numberOfItemsToAdd = 100;
+const defaultNumberOfItemsToAdd = 100;
 const defaultLanguage = "en";
 
+let numberOfItemsToAdd = defaultNumberOfItemsToAdd;
+
+export function handleComplexityChange() {
+    numberOfItemsToAdd = Math.abs(Math.round(defaultNumberOfItemsToAdd * params.complexity));
```

Comment:
Not sure why I suggested `Math.abs`, of course I meant `Math.round` 😅 

`Math.abs` is probably not needed actually, this can't be negative I guess?

Reply:
fair, we do check that in the params parsing already.
removing it.